Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java

```diff
@@ -78,34 +80,58 @@ public String getContent() {
     try {
       content = (String) this.jsonObject.get("content");
       Document jsoupDoc = Jsoup.parse(content);
-      content = jsoupDoc.text();
+      content = jsoupDoc.toString();
     } catch (Exception e) {
       Log.e("getContent_Error:", e.getMessage(), e);
       content = null;
     }
     return content;
   }
 
-  public Integer getWordCount() {
-    Integer wordCount = 0;
+  public int getWordCount() {
+    int wordCount;
     try {
-      wordCount = (Integer) this.jsonObject.get("word_count");
+      long longWordCount = (long) this.jsonObject.get("word_count");
+      wordCount = (int) longWordCount;
```

Comment:
Is it possible to combine those statements into `wordCount = (int) this.jsonObject.get("word_count");`?